lib/bolt/application.rb

```diff
@@ -258,7 +258,7 @@ def add_module(name, outputter)
                     config.project.puppetfile,
                     config.project.managed_moduledir,
                     config.project.project_file,
-                    config.module_install)
+                    @plugins.resolve_references(config.module_install))
```

Comment:
If we resolve here, isn't the whole `module_install` config setting or any subkeys also pluggable? I think that's totally fine, just want to make sure that that's known, and we should also update the data in `options.rb` for those options

Reply:
Technically, they can be. They still currently error when validating config, though, since only `module-install.forge.authorization_key` is marked as pluggable.

Reply:
Ah, ok :+1: